pkg/alertmanager/alertmanager_metrics.go

```diff
@@ -209,6 +212,18 @@ func newAlertmanagerMetrics() *alertmanagerMetrics {
 			"cortex_alertmanager_notification_rate_limited_total",
 			"Total number of rate-limited notifications per integration.",
 			[]string{"user", "integration"}, nil),
+		insertAlertFailures: prometheus.NewDesc(
+			"cortex_alertmanager_alerts_insert_failures_total",
+			"Total number of failures to store alert due to hitting alertmanager limits.",
+			[]string{"user"}, nil),
+		alertsLimiterAlertsCount: prometheus.NewDesc(
+			"cortex_alertmanager_alerts_limiter_current_alerts_count",
```

Comment:
[nit] Gauges typically don't have the `_count` suffix.

```suggestion
			"cortex_alertmanager_alerts_limiter_current_alerts",
```